.github/change_filters.yml

```diff
@@ -3,4 +3,3 @@ docs:
   - 'changelog/*'
   - 'CHANGELOG.mdx'
   - 'Makefile'
-  - '.github/workflows/documentation.yml'
```

Comment:
can we make sure the `change_filters.yml` file is removed? I think I asked about it in the other PR :)

Reply:
(and it needs to be removed in `3.3.x` too)